doc/develop/tools/clion.rst

```diff
@@ -0,0 +1,267 @@
+
+
+Get CLion
+*********
+
+Download_ and install CLion.
+
+.. _Download: https://www.jetbrains.com/clion/download
+
+Initialize a new workspace
+**************************
+
+#. Before you start, make sure to prepare a Zephyr development environment:
+
+   * `Install dependencies`_
+
+      .. _Install dependencies: https://docs.zephyrproject.org/latest/develop/getting_started/index.html#install-dependencies
+
+   * `Get Zephyr and install Python dependencies`_
+
+      .. _Get Zephyr and install Python dependencies: https://docs.zephyrproject.org/latest/develop/getting_started/index.html#get-zephyr-and-install-python-dependencies
+
+   * `Install Zephyr SDK`_
+
+      .. _Install Zephyr SDK: https://docs.zephyrproject.org/latest/develop/getting_started/index.html#install-zephyr-sdk
+
+#. Run the following command to initialize the workspace folder
+   (``my-workspace``) and clone the example application into it:
+
+   .. code-block::
+
+      west init -m https://github.com/zephyrproject-rtos/example-application
+      --mr main my-workspace
+
+#. Update Zephyr modules:
+
+   .. code-block::
+
+      cd my-workspace
+      west update
+
+
+Open the project in CLion
+**************************
+
+#. In CLion, click **Open** on the Welcome screen or select **File | Open**
+   from the main menu.
+
+#. Locate the workspace folder (``my-workspace``) and select it.
+
+   Click **OK**.
+
+#. If prompted, click **Trust Project**.
+
+   See `this section`_ in CLion web help for more information on project
+   security.
+
+   .. _this section: https://www.jetbrains.com/help/clion/project-security.html#projects_security
+    </chapter>
+
+
+Create a CMake presets file
+***************************
+
+This presets_ file will activate the Python virtual environment for your Zephyr
+installation and will define the target board, the build directory,
+and the flashing tool.
+
+.. _presets: https://cmake.org/cmake/help/latest/manual/cmake-presets.7.html
+
+You can find more information on how CLion works with CMake presets on
+`this web help page`_.
+
+.. _this web help page: https://www.jetbrains.com/help/clion/cmake-presets.html
+
+
+#. In the **Project** view, locate the ``my-workspace\example-application\app``
+   folder.
+
+#. Right-click the ``app`` folder and select **New | File**.
+
+#. Name the file ``CMakeUserPresets.json``.
+
+#. Add the following code into it:
+
+   .. code-block::
+
+      {
+         "version": 3,
+            "cmakeMinimumRequired": {
+            "major": 3,
+            "minor": 24
+         },
+         "configurePresets": [
+            {
+               "name": "Zephyr-Debug",
+               "generator": "Ninja",
+               "binaryDir": "build",
+               "cacheVariables": {
+                  "BOARD": "nucleo_f302r8",
+                  "CMAKE_PROGRAM_PATH": "&lt;OPENOCD_INSTALLATION_PATH&gt;;&lt;HOME_PATH&gt;/zephyrproject/.venv/Scripts"
```

Comment:
Don't use &lt; / &gt; as it's already a code block, plus, please explain better in the paragraph that follows that <SOME_TAGS> need to be replaced by actual values, as otherwise the fact they are written in capital letters makes it look like it might automagically work for these could be existing environment variables.

Reply:
Removed &lt; / &gt; and capitalization. Below we instruct to customize that line, is it clear what should be changed there?